test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -726,6 +726,10 @@ func testResourceDelete(c *testContext) {
 		}
 		return true, nil
 	})
+	if err != nil {
+		c.t.Error(err)
```

Comment:
instead of reporting this err you'd need to store the error from the above code in a variable and report that.

lines 715 and 725.

Reply:
@neolit123 Thanks.
Refer to Line 670 and Line 740,the context handle this error

Reply:
are you suggesting that line 670 already has the same pattern of not handling this error?
the problem is that this will print an error message that is not very descriptive.

but ok, leave it like that.

Reply:
@neolit123  can you give some advie to deal with this error if you can.Thanks
plz

Reply:
the error is handled the same way in the code above, so it's OK.

Reply:
ok,thanks